doc/compressed_transactions.md

```diff
@@ -0,0 +1,49 @@
+2. Removing all excess white space and fillers with the use of Variable Integers.
+3. The use of compressed signatures and public key recovery upon decompression
+4. Replacing the 32 byte Txid with a block height and index for the transaction instead.
+
+## 2. Schema
+
+| Name                           | Value     | Description |
+| -------------------------------|-----------|-------------|
+| Control VarInt(CV)             | 1-2 Bytes | A Variable Integer containing information on the structure of the transaction. |
+| CV->Version                    | 2 Bits    | If the Version is less then 4 the Version is encoded directly, otherwise 0b00 signals the existence  of a future VarInt for the Version. |
+| CV->Input Count                | 2 Bits    | If the Input Count is less then 4 the Input Count is encoded directly, otherwise 0b00 signals the existence of a future VarInt for the Input Count. |
+| CV->Output Count               | 2 Bits    | If the Output Count is less then 4 the Output Count is encoded directly, otherwise 0b00 signals the existence of a future VarInt for the Output Count. |
+| CV->LockTime                   | 2 Bits    | 0x00 if the LockTime is zero, 0x01 if the LockTime has been shortened to the sixteen least significant bits, 0x02 if is not been shortened and signals the existence of a future VarInt for the LockTime. |
+| CV->Coinbase                   | 1 Bits    | Boolean to determine if this transaction is a Coinbase Transaction. |
+| Version VarInt                 | 1-4 Bytes | If the Version was greater then 3 then this is an Optional Variable Integer containing the Version. |
+| Input Count VarInt             | 1-4 Bytes | If the Input Count was greater then 3 then this is an Optional Variable Integer containing the Input Count. |
+| Output Count VarInt            | 1-4 Bytes | If the Output Count was greater then 3 then this is an Optional Variable Integer containing the Output Count. |
+| LockTime VarInt                | 1-4 Bytes | An Optional Variable Integer containing either the full LockTime or the shortened sixteen least significant bits. |
+| Input/Output Stream(IO/Stream) | ? Bytes   | A Variable Length Bit Stream padded to full Bytes, Contains metadata on the Inputs and Outputs of the Transaction. |
+| IO/Stream->Signature Signed    | 1 Bits    | Boolean to determine if this Input has been Signed. |
+| IO/Stream->Signature Compressed| 1 Bits    | Boolean to determine if this Input's Signature has been Compressed. |
+| IO/Stream->Txid Compressed     | 1 Bits    | Boolean to determine if this Input's Txid has been Compressed. |
+| IO/Stream->Standard Hash       | 1 Bits    | Boolean to determine if this Input's Signature Hash Type is (0x00 or 0x01) or something else. |
+| IO/Stream->Hash Type           | 1 Bits    | Boolean to determine if this Input's Signature Hash Type is 0x00 or 0x01. |
```

Comment:
In 61504d1a4d7ea545b1880686e426b1acae7d26f0:

Can drop this bit since 0 vs 1 is determined by the taproot/non-taproot of the output.